frontend/javascripts/types/api_flow_types.ts

```diff
@@ -756,7 +756,7 @@ export type ServerNode = {
   rotation: Point3;
   bitDepth: number;
   viewport: number;
-  resolution: number;
+  mag: number;
```

Comment:
Here is the root of the problem. The backend sends nodes with the field called mag and not resolution. See: https://github.com/scalableminds/webknossos/blob/8dec57826e253f3b1c2d17308e7d75d41fea26b1/webknossos-datastore/proto/SkeletonTracing.proto#L8-L19
By changing the type, TS should show all locations to change and the resulting code should be fine.